tests/test-cases/Group6-VIC-Machine/6-04-Create-Basic.md

```diff
@@ -1,50 +1,76 @@
 Test 6-04 - Verify vic-machine create basic function
 =======
 
-#Purpose:
+# Purpose:
 Verify vic-machine create basic connection variables, certificates, timeout, and all arguments after appliance-iso
 
-#References:
+# References:
 * vic-machine-linux create -h
 
-#Environment:
+# Environment:
 This test requires that a vSphere server is running and available
 
 
 
-#Test Cases: - Connection
+Image size
+=======
+
+## Create VCH - custom base disk
+1. Issue the following command:
+```
+vic-machine-linux create --name=${vch-name} --target=%{TEST_URL} --thumbprint=%{TEST_THUMBPRINT} --user=%{TEST_USERNAME} --bridge-network=%{BRIDGE_NETWORK} --external-network=%{EXTERNAL_NETWORK} --image-store=%{TEST_DATASTORE} --password=%{TEST_PASSWORD} --base-image-size=6GB ${vicmachinetls}
+```
+
+### Expected Outcome
+* VCH is deployed successfully
+* Container has correct disk size
+* Regression tests pass
 
-#Test Steps
+
+
+Connection
+=======
+
+## Create VCH - URL without user and password
 1. Create with vSphere URL in --target parameter, without --user and --password
 
-#Expected Outcome:
+### Expected Outcome
 * Command should fail for no user password available
 
 
-#Test Steps
+## Create VCH - URL without password
 1. Create with vSphere URL in --target parameter and --user provided, but without --password
 
-#Expected Outcome:
+### Expected Outcome
 * Command should promote interactive password input
 
 
-#Test Steps
-1. Create with vSphere URL with wrong DC appended as <ip>/WrongDC
+## Create VCH - target URL
+1. Create with vSphere URL and user password encoded in the same --target parameter
+```
+vic-machine-linux create --name=<VCH_NAME> --target="<TEST_USERNAME>:<TEST_PASSWORD>@<TEST_URL>" \
+    --image-store=<TEST_DATASTORE>
+```
+2. Run regression tests
 
-#Expected Outcome:
-* Command should fail for WrongDC is not found
+### Expected Outcome
+* Deployment succeed
+* Regression test pass
 
 
-#Test Steps
+## Create VCH - specified datacenter
 1. Prepare test env with multiple DC exists
 2. Create with vSphere URL with correct DC appended as <ip>/DC1
 
-#Expected Outcome:
+### Expected Outcome
 * Verify deployed successfully
 * Verify VCH is in correct DC through govc
 
 
-#Test Steps
+
+Parameters
```

Comment:
Not sure what the idea of this giant Parameters thing is in the middle of the file?